pkg/controllers/user/helm/common/common.go

```diff
@@ -219,3 +222,17 @@ func JailCommand(cmd *exec.Cmd, jailPath string) (*exec.Cmd, error) {
 	cmd.Env = jailer.WhitelistEnvvars(cmd.Env)
 	return cmd, nil
 }
+
+// escapeCommas will escape the commas in a string, unless helm would identify it as a list
+func escapeCommas(value string) string {
+	if len(value) == 0 {
+		return value
+	}
+	// as far as helm is concerned, an answer starting with '{' is a list
+	// commas in lists are not escaped for users as they are also used as delimiters
+	if string(value[0]) == "{" {
```

Comment:
no need to confirm an ending `}` and validate the input before returning? just assume user input is correct and move on? does helm break and send back proper errors if you pass in a list like this `{a,b,c` and don't close it?